src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,25 @@
+package lotto.domain;
+
+import java.util.Set;
+
+public class Lotto {
+    private static final int REQUIRED_NUMBERS = 6;
+    private final Set<Integer> numbers;
```

Comment:
```suggestion
    private static final int REQUIRED_NUMBERS = 6;
    
    private final Set<Integer> numbers;
```

인스턴스 변수와 클래스 변수를 구분지으면 더 깔끔한 것 같아요